README.md

```diff
@@ -8,20 +8,8 @@ The MMS SDVC is a collection of modules built on top of the Spring Framework and
 We suggest using Docker to set up PostgreSQL and Elasticsearch.  Installation 
 instructions are found here: [Docker documentation](https://docs.docker.com/)
 
-### Postgresql
-Install postgres (PostgreSQL) 9.6, instructions for Docker: [PostgreSQL with Docker](https://hub.docker.com/_/postgres)
-
-    docker run -d -e POSTGRES_PASSWORD=test1234 -e POSTGRES_USER=mmsuser -e POSTGRES_DB=mms -p 5432:5432 postgres:9-alpine
-    
-### or Mysql
-5.7 [Mysql Docker](https://hub.docker.com/_/mysql/)
-
-    docker run -d -e MYSQL_ROOT_PASSWORD=test1234 -e MYSQL_DATABASE=mms -p 3306:3306 mysql:5.7
-
-### Elasticsearch
-Install Elasticsearch 7.1.  If you use Docker instructions are available here: [Setting up Elasticsearch with Docker](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html)
-
-    docker run -d -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:7.1.1
+### Java SE Development Kit 11
+Installation instructions: [JDK-11 download](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
```

Comment:
so this is for a developer setup - you would still want the original instructions for postgres/mysql/elastic, since doing docker-compose is more for just running/getting the application up as quickly as possible to play with/running in ci.

I would say add a separate section before the "Developer Setup" for the docker-compose instructions (something like quick start) - for that the only thing that's needed is docker